acquia_lift.module

```diff
@@ -3175,3 +2756,56 @@ function acquia_lift_agent_clear_verified_status($agent_name = NULL) {
   }
   cache_set(ACQUIA_LIFT_AGENT_VERIFY_CACHE, $verification);
 }
+
+/**
+ * Get the option set where targeting rules are defined.
+ *
+ * The assumption here is that if you have multiple option sets then targeting
+ * is only defined on one of them as we currently do not support targeting for
+ * MVTs.
+ *
+ * @param $agent
+ *   A stdClass object representing the agent to get the option set for.
+ * @return stdClass|NULL
+ *   The option set to use for targeting rules or NULL if the agent has no option
+ *   sets.
+ */
+function acquia_lift_get_option_set_for_targeting($agent_name) {
+  $option_sets = personalize_option_set_load_by_agent($agent_name, TRUE);
+  if (empty($option_sets)) {
+    return NULL;
+  }
+  foreach ($option_sets as $option_set) {
+    if (!empty($option_set->targeting)) {
+      return $option_set;
+    }
+  }
+  return reset($option_sets);
+}
+
+/**
+ * Returns the list of test agents that are nested under the specified agent.
+ *
+ * @param $agent
+ *   The agent to get nested test agents for.
+ * @return array
+ *   An array of agent names.
+ */
+function acquia_lift_get_nested_tests($agent) {
+  if ($agent->plugin != 'acquia_lift_target') {
+    return array();
+  }
+  $option_set = acquia_lift_get_option_set_for_targeting($agent->machine_name);
+  if (empty($option_set) || empty($option_set->targeting)) {
+    return array();
+  }
+  $agents = array();
+  foreach ($option_set->targeting as $audience_name => $audience) {
+    if (isset($audience['osid'])) {
+      if ($option_set = personalize_option_set_load($audience['osid'])) {
+        $agents[] = $option_set->agent;
+      }
+    }
+  }
+  return $agents;
+}
```

Comment:
I found something!  Ok.... well the big red arrow in github found something.  Line break.